### DISCUSSION TITLE
运行相关问题

### DISCUSSION THREAD

**Discussion by Fc8cF** [mikumifa/biliTickerBuy#215] [提问专区]
我尝试运行了软件，但是当我运行的时候会有一个弹窗说包含潜在垃圾文件什么什么的，因为一直在尝试所以没有截图。然后过了一会儿exe文件不在文件夹里了（我也不知道哪儿去了），以有没有大佬能帮帮TT
图片是我文件夹现在的样子
![1](https://github.com/mikumifa/biliTickerBuy/assets/146006959/d9dfc887-2123-478b-b510-5e66140d97e5)

**Comment by mikumifa** [comment]
杀毒软件杀了
把电脑所有安全管家全关了

**Reply by Fc8cF** [reply]
好的谢谢，我试一下

**Reply by Fc8cF** [reply]
搞好啦，谢谢
